src/Forms/ValidatorList.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+use SilverStripe\ORM\ArrayList;
+use SilverStripe\ORM\ValidationResult;
+
+/**
+ * Class ValidatorList
+ *
+ * @package SilverStripe\Forms
+ */
+class ValidatorList extends Validator
+{
+    /**
+     * @var ArrayList|Validator[]
+     */
+    private $validators;
```

Comment:
Any particular reason why this is an `ArrayList` instead of just an array?

Reply:
I don't have any strong feelings about `ArrayList` vs `array`. I think I just wondered if maybe(?) an `ArrayList` could be more useful to someone somewhere.

I would be happy to change it.

Reply:
My preference is an array as `ArrayList` is intended for models and model-like data structures, which this isn’t. I’m struggling to think of a use case that would benefit from it being an `ArrayList` 😅 you can always manually wrap it in an `ArrayList` anyway